kernel/usage.c

```diff
@@ -272,6 +272,22 @@ int k_thread_runtime_stats_disable(k_tid_t  thread)
 
 	return 0;
 }
+
+int k_thread_runtime_stats_is_enabled(k_tid_t thread)
```

Comment:
```suggestion
bool k_thread_runtime_stats_is_enabled(k_tid_t thread)
```

Please use `bool` for boolean data types.